portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path

```diff
@@ -858,6 +858,11 @@
 	<td>//button[contains(@class,'ytp-large-play-button')]</td>
 	<td></td>
 </tr>
+<tr>
+    <td>BUTTON_PLUS_PRODUCT_VISIBILITY</td>
+    <td>//li[contains(@class,'nav-item')]//button[contains(@title,'${key_webContentTitle}')]</td>
+    <td></td>
+</tr>
```

Comment:
I believe you can remove it

Reply:
how the previous change didn't work, I kept.